tasks/gotest.py

```diff
@@ -741,7 +741,7 @@ def format_packages(ctx: Context, impacted_packages: set[str], build_tags: list[
     modules_to_test = {}
 
     for package in packages:
```

Comment:
Note for reviewers: had to do it in this branch to make the pre commit hook happy, I can take it away to a separate PR